lms/static/scripts/frontend_apps/components/dashboard/DashboardBreadcrumbs.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ArrowLeftIcon,
+  CaretRightIcon,
+  Link,
+} from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import { Link as RouterLink } from 'wouter-preact';
+
+type BreadcrumbLink = {
+  title: string;
+  href: string;
+};
+
+export type DashboardBreadcrumbsProps = {
+  links: BreadcrumbLink[];
+};
+
+function BreadcrumbLink({ title, href }: BreadcrumbLink) {
+  return (
+    <RouterLink href={href} asChild>
+      <Link underline="hover" variant="text-light" classes="truncate">
+        <ArrowLeftIcon className="inline-block md:hidden mr-1 align-sub" />
+        {title}
+      </Link>
+    </RouterLink>
+  );
+}
+
+export default function DashboardBreadcrumbs({
```

Comment:
Can you add a short JSDoc comment explaining what this is from the user's point of view.